t/base/protocols.t

```diff
@@ -1,20 +1,23 @@
 use strict;
 use warnings;
 use Test::More;
-plan tests => 6;
 
 use LWP::UserAgent;
+plan tests => 8;
```

Comment:
I tend to agree with you.  However, after some discussion in #lwp this is preferable to requiring at least version 0.88 of Test::More.

I'll be happy to update things if we're willing to make that concession.